server/src/services/metadata.service.spec.ts

```diff
@@ -439,7 +439,7 @@ describe(MetadataService.name, () => {
       });
       cryptoRepository.hashSha1.mockReturnValue(randomBytes(512));
       assetMock.getByChecksum.mockResolvedValue(null);
-      assetMock.create.mockResolvedValue(assetStub.livePhotoMotionAsset);
+      assetMock.create.mockImplementation((asset) => Promise.resolve({ ...assetStub.livePhotoMotionAsset, ...asset }));
```

Comment:
AFAICT the create function is only called once, correct? Why can't we use `mockResolvedValue` then, given that we already know which asset we pass in.